src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -465,6 +473,41 @@ public function testDecodeEmptyXml()
         $this->encoder->decode(' ', 'xml');
     }
 
+    private function getXmlPISource()
```

Comment:
I would even get rid of this function and put the doc in `testDecodeXMLWithProcessInstruction`.